quay-pruner/config/definitions/quay-pruner.yml

```diff
@@ -0,0 +1,48 @@
+- scm:
+    name: ceph-build
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-build.git
+          branches:
+            - origin/master
+          browser-url: https://github.com/ceph/ceph-build
+          timeout: 20
+          basedir: "ceph-build"
+
+
+- job:
+    name: quay-pruner
+    node: small
+    project-type: freestyle
+    defaults: global
+    display-name: 'Quay: prune container images'
+    concurrent: true
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          artifact-days-to-keep: 15
+
+    triggers:
+      - timed: '@daily'
+
+    scm:
+      - ceph-build
+
+
+    builders:
+      - shell:
+          !include-raw:
+            - ../../build/build
+
+    wrappers:
+      - inject-passwords:
+          global: true
+          mask-password-params: true
+      - credentials-binding:
+          - text:
+              credential-id: quay-api-token
+              variable: QUAYTOKEN
```

Comment:
Can we add some comment here about this `quay-api-token`? Like which account does this belong to? When does it expire? How do we refresh it?

Reply:
Well, that's a good question.  All of this is currently "my personal account", and I need to understand how we genericize and record it.  Currently, of course, it's a global credential in Jenkins.

Reply:
The same applies to the login used in ceph-build/ceph-container, and it's an outstanding item to straighten that out.  When it first came up I was thinking waiting for you to return would be best, but now's the time.

Reply:
Let's keep discussing this offline